### PR TITLE
Api clean

### PR DESCRIPTION
**Pull request by Zerowhizkey**
Third times the charm ;) <3

### CODE REVIEW

File: src/routes/Profile/Profile.tsx [hunk 1]
```diff
@@ -40,26 +40,28 @@ const User = () => {
     const [vegan, setVegan] = useState(false);
     const [vegetarian, setVegetarian] = useState(false);
 
-    const userId = user.id;
+    // const userId = user.id;
```
Comment (linda-94-andersson) [inline src/routes/Profile/Profile.tsx:43]: Remove this if not used thank you! :)

**Review by linda-94-andersson — CHANGES_REQUESTED** [review]
Awesome work! Please remove one line <3

File: src/routes/Profile/Profile.tsx (hunk 1 above)
Comment (Zerowhizkey) [reply src/routes/Profile/Profile.tsx]: Done

**Review by linda-94-andersson — APPROVED** [review]
NICE!

**Pull request merged**